astronomer/providers/amazon/aws/triggers/redshift_cluster.py

```diff
@@ -58,20 +58,16 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                 response = await hook.resume_cluster(cluster_identifier=self.cluster_identifier)
                 if response:
                     yield TriggerEvent(response)
-                    return
                 else:
                     error_message = f"{self.task_id} failed"
                     yield TriggerEvent({"status": "error", "message": str(error_message)})
```

Comment:
```suggestion
                    yield TriggerEvent({"status": "error", "message": error_message})
```
Not your change, but error_message on line 62 is already a string.